src/open_inwoner/js/components/form/LoginForm.js

```diff
@@ -0,0 +1,83 @@
+export class LoginFormFocus {
+  static selector = '#login-form'
+
+  constructor(node) {
+    this.node = node
+    this.usernameInput = node.querySelector('input[name="username"]')
+    this.loginFormColumn = document.getElementById('column__login-form')
+    this.emailToggleParent = document.getElementById('column__email-toggle')
+
+    this.removeAutofocusAndFocus()
+    this.hideLoginFormOnLoad()
+    this.addEmailToggleListener()
+    this.activateTabFromHash()
+  }
+
+  removeAutofocusAndFocus() {
+    if (this.usernameInput) {
+      this.usernameInput.removeAttribute('autofocus')
+      this.usernameInput.blur()
+    }
+  }
+
+  hideLoginFormOnLoad() {
+    if (this.loginFormColumn) {
+      this.emailToggleParent.setAttribute('aria-expanded', 'false')
+      this.loginFormColumn.classList.add('hide')
+    }
+  }
+
+  addEmailToggleListener() {
+    if (this.emailToggleParent) {
+      const emailToggleParents =
+        this.emailToggleParent.querySelectorAll('.link')
+      emailToggleParents.forEach((link) => {
+        link.addEventListener('click', (event) => {
+          event.preventDefault()
+          this.emailToggleParent.classList.add('hide')
+          this.toggleLoginFormVisibility()
+        })
+      })
+    }
+  }
+
+  toggleLoginFormVisibility() {
+    if (this.loginFormColumn) {
+      this.loginFormColumn.classList.toggle('hide')
+      this.usernameInput.focus()
+    }
+  }
+
+  activateTabFromHash() {
+    const hash = window.location.hash
+    const particulierLink = document.querySelector(
+      '.tab__header[href="/accounts/login/#particulier"]'
+    )
+    const zakelijkLink = document.querySelector(
+      '.tab__header[href="/accounts/login/#zakelijk"]'
```

Comment:
Feedback for later: Django URL's are technically dynamic so we don't want hardcoded URL's in the Javascript (or HTML, CSS etc). This is why there are `{% url "login" %}` in the templates (to `reverse()` the URLs from whatever is in the urls.py's).
 
So these Link selectors should be different and not use the href= but a class or id (maybe chain of the Tab if they wrap the Link)

Reply:
OK, @Bartvaderkin I will have to create new id's/selectors here because the tabs are not inside those panels-with-id's; 
they are elsewhere in the HTML structure.